Sources/FluentKit/Query/Builder/QueryBuilder+Join.swift

```diff
@@ -12,6 +12,72 @@ extension QueryBuilder {
         self.join(Foreign.self, filter.foreign, to: Local.self, filter.local , method: method)
     }
 
+    /// This will join a foreign table based on a `ParentProperty`relation
```

Comment:
What about "based on a `@Parent` relation"? I'm not sure people will know `ParentProperty` == `@Parent`.